lib/tictactoe.rb

```diff
@@ -32,7 +30,7 @@ def play_turn
   end
 
   def play_game
-    while game.outcome(board).status == :in_progress
+    while board.outcome.status == :in_progress
```

Comment:
Maybe it makes sense to make this a metho don the board?
```suggestion
    while board.in_progress?
```

Reply:
More concise.  Cool.